src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -79,6 +81,26 @@ const StateModel = Backbone.Model.extend({
 
     return filters[this.id];
   },
+  setSelected(model) {
```

Comment:
This and `removeSelected` could be consolidated into 1 method if it also accepts an `isSelected` boolean. Thoughts?